hs-bindgen/fixtures/attributes/attributes/bindingspec.yaml

```diff
@@ -5,7 +5,7 @@ target: x86_64-pc-linux-musl
 hsmodule: Example
 ctypes:
 - headers: attributes/attributes.h
-  cname: struct __sFILE
+  cname: FILE
```

Comment:
@TravisCardwell In a case like this where we squash a typedef around a struct, and then use the name of the typedef _for_ the struct, the generated binding spec now refers to the typedef instead of the struct. I think this is okay (in fact, better).